src/main/scala/fi/aalto/cs/apluscourses/intellij/actions/REPLAction.scala

```diff
@@ -0,0 +1,64 @@
+package fi.aalto.cs.apluscourses.intellij.actions
+
+import com.intellij.execution.{RunManager, RunManagerEx, RunnerAndConfigurationSettings}
+import com.intellij.openapi.actionSystem.{AnActionEvent, CommonDataKeys}
+import com.intellij.openapi.module.{Module, ModuleUtilCore}
+import com.intellij.openapi.vfs.VirtualFile
+import org.jetbrains.annotations.{NotNull, Nullable}
+import org.jetbrains.plugins.scala.console.actions.RunConsoleAction
+import org.jetbrains.plugins.scala.console.configuration.ScalaConsoleRunConfiguration
+
+import scala.collection.JavaConverters._
```

Comment:
this is for scala **2.12**, as I had to downgrade from **2.13**, because some terrible things started to happen during the compilation phase if the version was not the same as the one for [JB's Scala Plugin](https://plugins.jetbrains.com/plugin/1347-scala)